webviews/components/sidebar.tsx

```diff
@@ -166,6 +166,7 @@ function Label(label: ILabel & { canDelete: boolean }) {
 				backgroundColor: labelColor.backgroundColor,
 				color: labelColor.textColor,
 				borderColor: `${labelColor.borderColor}`,
+				paddingRight: canDelete ? '2px' : '8px'
```

Comment:
Is this to handle PRs where you can't remove the labels yourself? I.e. they don't feature an icon?

Reply:
Before/after screenshot would be helpful for me here